### DISCUSSION TITLE
Restarting Simulations with Different Processor Counts in MOOSE

### DISCUSSION THREAD

**Discussion by shrituntunroy** [idaholab/moose#28506] [Q&A Modules: General]
Hi MOOSE Team,

I’m a new MOOSE user currently learning how to work with the framework.

I have a question about restarting a simulation in a MOOSE-based application. Specifically, I’d like to know if it’s possible to restart a simulation using a different number of processors than the original run.

I’m aware of the --recover option using checkpoints, which works well when the number of processors remains the same across both runs. However, I’ve encountered issues when trying to restart with a different number of processors. Could you provide guidance on how to handle this situation?

Thank you for your help.

**Comment by GiudGiud** [comment]
Hello

We do not support this using checkpoint.
You can attempt N-to-M restarts using exodus files (either as the main mesh file, or using a SolutionUserObject), but there are a number of items (stateful properties, older variable values) that will not get restarted.
https://mooseframework.inl.gov/application_usage/restart_recover.html

Guillaume
